src/utils/results.ts

```diff
@@ -4,7 +4,7 @@ export type Failure = { success: false; message: string; errors?: any };
 export type Result<T> = Success<T> | Failure;
 
 // helpers
-export const success = <T>(data: T): Success<T> => ({ success: true, exception: data });
+export const success = <T>(exception: T): Success<T> => ({ success: true, exception });
```

Comment:
Naming consistency. We want to update this whole type to use `data` instead of `exception`, but we will make these changes at the end following all the merges (fewer conflicts hopefully).